releasenotes/notes/botocore-46f5bf04f64777b4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    botocore: added `invoke_with_legacy_context` configuration setting, which is disabled by default. This configuration setting is to preserve a legacy behavior in which, when tracing a direct lambda invocation, trace context was propagated by storing it in a `_datadog` object in `context.clientContext.custom`. This must be set to True if you are invoking Lambda functions instrumented with datadog-lambda-python < v41 or datadog-lambda-js < v3.57.0. However, if set to True, invocations to Go or Java Lambda functions may fail.
```

Comment:
I'd probably trim this down as well. From the user standpoint they just need to know that context management support has been added for newer lambda libraries and that there is a legacy configuration option in-case they wish to run an older version.

:+1: for mentioning the specific versions of each library.

Would also be good to explicitly show the configuration options, something like:

```
Legacy support for older libraries is available with ddtrace.config.botocore.invoke_with_legacy_context = True or by setting the environment variable DD_BOTOCORE_INVOKE_WITH_LEGACY_CONTEXT=true
```

Reply:
I'd like to see it explicitly mentioned that the propagation will break for users using `datadog-lambda-python < v41 or datadog-lambda-js < v3.57.0` first and then mention the introduction of the legacy setting that can be used to support these older versions.